include/tables/XAttrHelper.h

```diff
@@ -0,0 +1,138 @@
+/*
```

Comment:
not sure if we need to extract these methods into a helper class. we can discuss tomorrow